### PR TITLE
Error handling for http get and post based off error code int value

### PR DESCRIPTION
**Pull request by abatisto**

### CODE REVIEW

File: ui/UI/controller/AccountCreation.swift [hunk 1]
```diff
@@ -184,50 +198,50 @@ class AccountCreation: UIViewController, UITableViewDelegate, UITableViewDataSou
...
+            
+            if let cell = tableView.cellForRow(at: indexPath) {
+                /*
+                 If there is already a checkmark, remove checkmark from the cell, set checked = false
+                 in checked array and remove genre from selected genres array
+                 */
+                if cell.accessoryType == .checkmark {
+                    cell.accessoryType = .none
+                    checkedCellsInstruments[indexPath.row] = false
+                    print("You unselected \(instrument)!")
```
[the remark this answers is not in the document]
Comment (bmckalla) [reply ui/UI/controller/AccountCreation.swift]: And not necessarily very useful in general unless for debugging (in which it shouldn't be in the app code anyway)

**Comment by bmckalla** [comment]
FYI you don't have to submit another pull request to change stuff, just make commits and push to the branch that this pull request was made under.

**Comment by abatisto** [comment]
Yeah my bad forgot I had already submitted a pull request the other day. Definitely should remove the print to the console for selecting and unselecting genres and instruments that was just something I did when putting the genre picker together. And as for the createAlert function call it should look like this:


DispatchQueue.main.async {

     self.createAlert(title: "title", message:"message")

}


I'm not really the best with optionals either but I'll take a closer look and try to figure out that part of the code.

________________________________

**Comment by bmckalla** [comment]
Sound good, thanks

**Pull request merged**